### PR TITLE
Update `selector-pseudo-element-colon-notation` to use `double`

### LINKED ISSUE #165: Update `selector-pseudo-element-colon-notation` to use `double`
As [IE 8 support has been dropped](https://make.wordpress.org/core/2017/04/23/target-browser-coverage/) from WordPress we should now switch to using`::` double colons from `:` single colons for the `selector-pseudo-element-colon-notation` rule

See also https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements

### PR DESCRIPTION
**Pull request by thebrandonallen**
Fixes #165

### CODE REVIEW

File: __tests__/selectors-invalid.css [hunk 1]
```diff
@@ -25,3 +25,7 @@ input[type='text'] { /* Should be [type="text"] */
 .selectorA {
 	color: #000;
 }
+
+.selector:hover {
+  color: #000;
```
Comment (ntwb) [inline __tests__/selectors-invalid.css]: Indent using a single tab instead of two spaces please

File: __tests__/selectors-invalid.scss [hunk 2]
```diff
@@ -25,3 +25,10 @@ p {
 		margin: 0;
 	}
 }
+
+p {
+
+	& .class:hover {
+		margin: 0;
```
Comment (ntwb) [inline __tests__/selectors-invalid.scss]: Indent using a single tab instead of two spaces please

File: __tests__/selectors-valid.css [hunk 3]
```diff
@@ -5,3 +5,7 @@
 input[type="text"] {
 	line-height: 1.1;
 }
+
+.selector::hover {
+  color: #000;
```
Comment (ntwb) [inline __tests__/selectors-valid.css]: Indent using a single tab instead of two spaces please

File: __tests__/selectors-valid.scss [hunk 4]
```diff
@@ -27,3 +27,11 @@ p {
 		margin: 0;
 	}
 }
+
+p {
+
+	&,
+	.foo::hover {
+		color: #000;
```
Comment (ntwb) [inline __tests__/selectors-valid.scss]: Indent using a single tab instead of two spaces please

File: __tests__/selectors-valid.css [hunk 5]
```diff
@@ -5,3 +5,7 @@
 input[type="text"] {
 	line-height: 1.1;
 }
+
+.selector::hover {
```
Comment (ntwb) [inline __tests__/selectors-valid.css]: `:hover` is [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes), can you change this to `::after` which is a [psuedo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) which is what this rule change is testing.

Double colons are _only_ valid for _psuedo-elements_, single colons remain for _psuedo-classes_

File: __tests__/selectors-valid.scss [hunk 6]
```diff
@@ -27,3 +27,11 @@ p {
 		margin: 0;
 	}
 }
+
+p {
+
+	&,
+	.foo::hover {
```
Comment (ntwb) [inline __tests__/selectors-valid.scss]: Likewise, change this to `::after` please

**Comment by ntwb** [comment]
Thanks @thebrandonallen, can you make the changes I've suggested above please 😄 

I noticed the ~50 or so errors when running the tests, pretty confusing I'm sure, I've removed the vast majority of those in #176, with some luck the tests should pass (these can be run via `npm test`)

**Comment by thebrandonallen** [comment]
Looks like `.editorconfig` is set to 2 spaces for indentation. Either way, I should have noticed 🤦‍♂️ 

Also, just learned `pseudo-class` vs `pseudo-element`. Guess I've always just seen "pseudo" and moved on.

Changes pushed.

**Comment by GaryJones** [comment]
@ntwb Is this good to go now?

**Comment by ntwb** [comment]
@thebrandonallen Can you change the option here on this PR to "Allow edits from maintainers" please, I've fixed the merge commits but unable to push these changes.

https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/#enabling-repository-maintainer-permissions-on-existing-pull-requests

**Comment by ntwb** [comment]
Nevermind, fixed and merged this PR into #189 

Thanks again @thebrandonallen and sorry this took so long to get too

**Pull request closed without merging**